kmod/src/scoutfs_trace.h

```diff
@@ -2031,8 +2031,8 @@ TRACE_EVENT(scoutfs_quorum_loop,
 		__field(int, vote_for)
 		__field(unsigned long, vote_bits)
 		__field(unsigned long, vote_count)
-		__field(unsigned long long, timeout_sec)
```

Comment:
The trace events use basic types because of the print format using %llu below.  It'd trigger warnings if the types ever went out of alignment, this can use an unsigned long long nsecs and then print that.